tests/backend/topology/test_von_neumann.py

```diff
@@ -49,18 +49,60 @@ def test_compute_position_return_values(swarm, bounds):
 def test_neighbor_idx(swarm, p, r):
     """Test if the neighbor_idx attribute is assigned"""
     topology = VonNeumann()
-    p = topology.compute_gbest(swarm, p=p, r=r)
+    topology.compute_gbest(swarm, p=p, r=r)
     assert topology.neighbor_idx is not None
 
 
 @pytest.mark.parametrize("m", [i for i in range(9)])
 @pytest.mark.parametrize("n", [i for i in range(10)])
 def test_delannoy_numbers(m, n):
-    expected_values = np.array([1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 25,
-                                41, 61, 63, 85, 113, 129, 145, 181, 231,
-                                321, 377, 575, 681, 833, 1159, 1289,
-                                1683, 2241, 3649, 3653, 5641, 7183,
-                                8989, 13073, 19825, 40081, 48639, 75517,
-                                108545, 22363, 224143, 265729, 598417])
+    expected_values = np.array(
+        [
+            1,
+            3,
+            5,
+            7,
+            9,
+            11,
+            13,
+            15,
+            17,
+            19,
+            25,
+            41,
+            61,
+            63,
+            85,
+            113,
+            129,
+            145,
+            181,
+            231,
+            321,
+            377,
+            575,
+            681,
+            833,
+            1159,
+            1289,
+            1683,
+            2241,
+            3649,
+            3653,
+            5641,
+            7183,
+            8989,
+            13073,
+            19825,
+            40081,
+            48639,
+            75517,
+            108545,
+            22363,
+            224143,
+            265729,
+            598417,
+        ]
```

Comment:
This one doesn't look quite nice. Maybe we should bump our default spacing to `88`. Did you ran black here with `79` as default? Can you run black again?